README.md

```diff
@@ -1,10 +1,107 @@
 <p align="center">
-<img
-    src="https://neo-cdn.azureedge.net/images/neo_logo.svg"
-    width="250px">
+  <a href="https://neo.org/">
+      <img
+      src="https://neo3.azureedge.net/images/logo%20files-dark.svg"
+      width="250px" alt="neo-logo">
+  </a>
 </p>
 
+<h3 align="center">Neo Blockchain</h3>
+
+<p align="center">
+   A modern distributed network for the Smart Economy.
+  <br>
+  <a href="https://docs.neo.org/docs/en-us/index.html"><strong>Documentation »</strong></a>
+  <br>
+  <br>
+  <a href="https://github.com/neo-project/neo"><strong>Neo</strong></a>
+  ·
+  <a href="https://github.com/neo-project/neo-vm">Neo VM</a>
+  ·
+  <a href="https://github.com/neo-project/neo-plugins">Neo Plugins</a>
+  ·
+  <a href="https://github.com/neo-project/neo-devpack-dotnet">Neo DevPack</a>
+  ·
+  <a href="https://github.com/neo-project/neo-node">Neo Node</a>
+</p>
 <p align="center">
+  <a href="https://twitter.com/neo_blockchain">
+      <img
+      src=".github/images/twitter-logo.png"
+      width="25px">
+  </a>
+  &nbsp;
+  <a href="https://medium.com/neo-smart-economy">
+      <img
+      src=".github/images/medium-logo.png"
+      width="23px">
+  </a>
+  &nbsp;
+  <a href="https://neonewstoday.com">
+      <img
+      src=".github/images/nnt-logo.jpg"
+      width="23px">
+  </a>
+  &nbsp;  
+  <a href="https://t.me/NEO_EN">
+      <img
+      src=".github/images/telegram-logo.png"
+      width="24px" >
+  </a>
+  &nbsp;
+  <a href="https://www.reddit.com/r/NEO/">
+      <img
+      src=".github/images/reddit-logo.png"
+      width="24px">
+  </a>
+  &nbsp;
+  <a href="https://discord.io/neo">
+      <img
+      src=".github/images/discord-logo.png"
+      width="25px">
+  </a>
+  &nbsp;
+  <a href="https://www.youtube.com/channel/UCl1AwEDN0w5lTmfJEMsY5Vw/videos">
```

Comment:
We should rename the youtube channel.

Reply:
@shargon  do you have the correct url?

Reply:
No, I think that there are no other one, but we should change it

Reply:
> No, I think that there are no other one, but we should change it

This is the official youtube channel so far. Do you think that we should have more development related videos to be linked?

Reply:
I know but I think that we have enough subscriptions to be able to choose a name, and it's more user friendly.

Reply:
I didn't get you well. The youtube channel is named like the other official social media channel, called "Neo Smart Economy". Do you think we should change the name? Or is it because now it's linked to the video list instead of the home page?

Reply:
Yes, but we don't have an url like this youtube.com/neo-smart-economy

Reply:
Done. You can replace the youtube link with: https://www.youtube.com/neosmarteconomy @lock9